experimental/sdata/tabular/testdata/histogram-multiple.golden.txt

```diff
@@ -0,0 +1,22 @@
+🌟 This was machine generated.  Do not edit. 🌟
+
+Frame[0] {
+    "type": "table"
```

Comment:
```suggestion
    "type": "histogram-buckets"
```
(or something like that) -- I think a key par of any defined type needs to be a fixed key for "type" -- this way the visualizations can know how to read the result without too much effort